### ISSUE TITLE
Optional version without Google Nodes

### ISSUE THREAD

**Issue by cookiechainsawxx** [AlekPet/ComfyUI_Custom_Nodes_AlekPet#91]
Good morning sir, would it be possible to have a version without "anything Google" related? I can't seem to make it work by removing those nodes manually. 

Thanks in advance, good day to you.

**Comment by AlekPet** [comment]
Yes, I think it's possible, you can make a separate branch. Although it seems like it's not difficult to delete. You need to delete the GoogleTransaltor folder and fix the __init__.py file.

**Comment by cookiechainsawxx** [comment]
Thanks I'll look into that.

**Comment by AlekPet** [comment]
> Thanks I'll look into that.

Hi. I thought about it and finished my old script **`__init__.py`**, which in the **config.json** file allows you to enable and disable certain nodes and the entire node. For everything to work, you first need to enable `"check_enabled_nodes": true` in **config.json**, and only then enable or disable nodes. https://github.com/AlekPet/ComfyUI_Custom_Nodes_AlekPet/tree/new_init_config_disabled

**Comment by cookiechainsawxx** [comment]
I already make it work by just editing the _init_.py  as suggested earlier and it runs without any error (that said, thank for the trouble), I only had one more doubt about POSE:
For POSE to work properly I need IP Adapter right?  "https://github.com/cubiq/ComfyUI_IPAdapter_plus". 
I haven't try to send the pose node input directly into the workflow, instead I connect it to the Painter node for quick doddle drawing and noticed that it won't be registered as a normal draw (that is a stick figure made with the pencil tool), and if send over and image it would be interpreted as colors and made into clothe-patters. 

Just wandering what was the implementation idea for this node into the core node that is painter, also having a graphical glitch when pose node are present on the workflow their canvas won't resize when doing zoom in and out and overlap with the UI of the workflow.

**Comment by AlekPet** [comment]
Hi. As for pose, I didn't quite understand what kind of graphic glitch, but at the output, both pose and psinter have an image. I know that when changing the scale in the browser, control points are displayed incorrectly in the selection mode. For pose, you need controlnet and the openpose preprocess type, to set the pose on the image, and ipadapter for something else. Painter can also be used like this: https://youtu.be/pHGyQSr5Yj4?si=U2zJ_Myna0njacpH

**Comment by cookiechainsawxx** [comment]
Many thanks, sir, painter node set to be one of my all time favorites.

**Comment by Sorry-Im-Stupid** [comment]
Could I get a version of this with just the ability to paint in the UI?
I can't even find the config.json file for this addon.

**Comment by AlekPet** [comment]
> Could I get a version of this with just the ability to paint in the UI?
> I can't even find the config.json file for this addon.

As I understand it, you only need Painter, then you need to go here and download two files `__init__.py` `config.json` and put them in the folder with my nodes and in the file `config.json` disable everything except PainterNode. The config should look like this:
```json
{
  "nodes_settings": {
    "ArgosTranslateNode": {
      "active": false,
      "nodes": {
        "ArgosTranslateCLIPTextEncodeNode": true,
        "ArgosTranslateTextNode": true
      }
    },
    "ChatGLMNode": {
      "active": false,
      "nodes": {
        "ChatGLM4TranslateCLIPTextEncodeNode": true,
        "ChatGLM4TranslateTextNode": true
      }
    },
    "DeepTranslatorNode": {
      "active": false,
      "nodes": {
        "DeepTranslatorCLIPTextEncodeNode": true,
        "DeepTranslatorTextNode": true
      }
    },
    "ExtrasNode": {
      "active": false,
      "nodes": {
        "PreviewTextNode": true,
        "HexToHueNode": true,
        "ColorsCorrectNode": true
      }
    },
    "GoogleTranslateNode": {
      "active": false,
      "nodes": {
        "GoogleTranslateCLIPTextEncodeNode": true,
        "GoogleTranslateTextNode": true
      }
    },
    "PainterNode": {
      "active": true,
      "nodes": {
        "PainterNode": true
      }
    },
    "PoseNode": {
      "active": false,
      "nodes": {
        "PoseNode": true
      }
    },
    "IDENode": {
      "active": false,
      "nodes": {
        "IDENode": true
      }
    }
  },
  "global_settings": {
    "check_enabled_nodes": true
  }
}
```
